### PR TITLE
Minor review of FAQ

### PR DESCRIPTION
**Pull request by justinegeffen**
Minor terminology changes. Following up on Guest Accounts FAQ question/answer content.

### CODE REVIEW

**Comment by justinegeffen** [comment]
Some very minor changes have been made so far and I have pinged @wiersgallak re: the guest accounts question. 

Other aspects I'd like to look into are Whether deactivated users are affected by guest accounts and cross-check the feature matrix.

File: source/overview/faq.rst [hunk 1]
```diff
@@ -343,15 +344,15 @@ Mattermost is an open source, self-hosted alternative to proprietary SaaS servic
...
 
 Use Cases
 ------------------
 
 Does Mattermost support external guests?
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Not yet. Guest accounts are a feature planned for Enterprise Edition. A pricing model proportional to a fraction of the value this new feature provides is expected. 
+Yes, the Guest Account feature allows organizations to collaborate with external users within a controlled environment. See the _`Guest Accounts (Beta) documentation <https://docs.mattermost.com/deployment/guest-accounts.html>`_ for more information.
```
Comment (amyblais) [inline source/overview/faq.rst]:
```suggestion
Yes, the Guest Account feature allows organizations to collaborate with external users within a controlled environment. See the _`Guest Accounts (Beta) documentation <https://docs.mattermost.com/deployment/guest-accounts.html>`__ for more information. 
```

Comment (amyblais) [reply source/overview/faq.rst]: (The link isn't rendering correctly)

File: source/overview/faq.rst [hunk 2]
```diff
@@ -437,30 +438,30 @@ The following chart highlights the end user features of Mattermost and their sup
 High Trust Questions
 ----------------------
 
-What are the trust benefits of Mattermost compared to 3rd party SaaS systems that let customers manage their own encryption keys?
+What are the trust benefits of Mattermost compared to third-party SaaS systems that let customers manage their own encryption keys?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  
-**Encryption doesn't mean a 3rd party SaaS vendor can't read your data**. A 3rd party vendor who provides encryption keys to the database that stores a customer's data at rest may still be able to read a customer's data while its in transit.
+**Encryption doesn't mean a third-party SaaS vendor can't read your data**. A third party vendor who provides encryption keys to the database that stores a customer's data at rest may still be able to read a customer's data while its in transit.
```
Comment (amyblais) [inline source/overview/faq.rst]:
```suggestion
**Encryption doesn't mean a third-party SaaS vendor can't read your data**. A third-party vendor who provides encryption keys to the database that stores a customer's data at rest may still be able to read a customer's data while its in transit.
```

Comment (amyblais) [reply source/overview/faq.rst]: (For consistency)

File: source/overview/faq.rst [hunk 3]
```diff
@@ -437,30 +438,30 @@ The following chart highlights the end user features of Mattermost and their sup
...
  
 In contrast, Mattermost is hosted by the customer. Not only can data be encrypted at rest and in-transit with keys generated by the customer (which no vendors ever touch), unencrypted data for search and mobile notifications is handled by systems under your IT team's control. 
 
 If it’s unclear from the vendor’s documentation whether or not your data can be read, ask them directly. 
 
 **Moreover, high trust enterprises need more than encryption - they need privacy, total data ownership, auditability, and control of their infrastructure.**
  
-*Privacy* means a 3rd party service cannot monitor the identity, IP address, location or access patterns of your employees, nor their activity on your system, nor provide that information either intentionally through a court order (which you may never be informed about) or unintentionally through a data breach.
+*Privacy* means a third party service cannot monitor the identity, IP address, location, or access patterns of your employees, nor their activity on your system, nor provide that information either intentionally through a court order (which you may never be informed about) or unintentionally through a data breach.
```
Comment (amyblais) [inline source/overview/faq.rst]:
```suggestion
*Privacy* means a third-party service cannot monitor the identity, IP address, location, or access patterns of your employees, nor their activity on your system, nor provide that information either intentionally through a court order (which you may never be informed about) or unintentionally through a data breach.
```

File: source/overview/faq.rst (hunk 2 above)
Comment (justinegeffen) [reply source/overview/faq.rst]: Thanks, @amyblais. I was doing some reading around "third-party" vs. "third party" and it looks like I missed the mark on this instance and the one below. :)

**Pull request merged**